fvm/src/call_manager/default.rs

```diff
@@ -168,10 +168,11 @@ where
 
     fn with_transaction(
         &mut self,
+        read_only: bool,
```

Comment:
What's a reaonly transaction?

Kind of nonsensical, but i guess convenient in not having to change existing code.

Reply:
Yeah, it is kind of subtle... I'm effectively just counting how many times I've entered "read-only" mode so I can exit when I get back to the top.